koku/masu/processor/aws/aws_report_parquet_summary_updater.py

```diff
@@ -121,9 +121,9 @@ def update_summary_tables(self, start_date, end_date):
                 accessor.populate_line_item_daily_summary_table_presto(
                     start, end, self._provider.uuid, current_bill_id, markup_value
                 )
-                accessor.populate_enabled_tag_keys(start, end, bill_ids)
+                # accessor.populate_enabled_tag_keys(start, end, bill_ids)
             accessor.populate_tags_summary_table(bill_ids, start_date, end_date)
-            accessor.update_line_item_daily_summary_with_enabled_tags(start_date, end_date, bill_ids)
+            # accessor.update_line_item_daily_summary_with_enabled_tags(start_date, end_date, bill_ids)
```

Comment:
We'll probably still want to figure out how to only populate the daily summary table with enabled tags since this is no longer being done. Can we run smoke tests on this to make sure this and the tag enablement for AWS still passes?

Reply:
Altered trino sql to handle tag filtering up-front for AWS.